test/forms/ConsignmentReferencesSpec.scala

```diff
@@ -42,28 +42,46 @@ class ConsignmentReferencesSpec extends BaseSpec {
 
   "Consignment References mapping" should {
     "return errors for empty fields" in {
-      val inputData = ConsignmentReferences(None, "", "")
+      val inputData = ConsignmentReferences(Some(""), "", "")
       val errors = ConsignmentReferences.form().fillAndValidate(inputData).errors
 
-      errors.length must be(2)
+      errors.length must be(3)
+      errors must contain(FormError("eori", "consignmentReferences.eori.error"))
       errors must contain(FormError("reference", "consignmentReferences.reference.empty"))
       errors must contain(FormError("referenceValue", "consignmentReferences.reference.value.empty"))
     }
 
     "return error for incorrect reference" in {
-      val inputData = ConsignmentReferences(Some("GB71757250450281160"), "Incorrect reference", "12345")
+      val inputData =
+        ConsignmentReferences(Some("GB71757250450281160"), "Incorrect reference", "5123456789-000-123ABC45$%^FIIIII")
       val errors = ConsignmentReferences.form().fillAndValidate(inputData).errors
 
-      errors.length must be(2)
+      errors.length must be(3)
```

Comment:
Assert this in a single line.  errors mustBe List(....)

Reply:
Addressed!